infrastructure-playbooks/purge-cluster.yml

```diff
@@ -343,6 +343,10 @@
     failed_when: false
     register: ceph_journal_partlabels
 
+  - name: see if encrypted partitions are present
+    command: blkid -t TYPE=crypto_LUKS -o value -s PARTUUID
```

Comment:
This might take a bit more time as I'm trying to do that with Ansible only so we don't need an ugly shell command...